apps/webpack.config.js

```diff
@@ -75,6 +84,46 @@ const nodePolyfillConfig = {
       process: 'process/browser',
       timers: 'timers-browserify',
     }),
+    new CircularDependencyPlugin({
+      // ignore everything in a build directory or mode_modules
```

Comment:
I would like to see some clarification here on why these things are both desirable to ignore; I assume it's simply because we won't have the ability to fix most circles detected in `node_modules` and that those same circles will also manifest in the build, but it would be nice to see that explicitly laid out.

Reply:
those directories are our of our source code control. If there's a circle that was produced from a build (shouldn't be possible) or an imported node_module (not our code), it's not something we should be worrying about or trying to fix, really. 

We could flag and report them to the maintainer to resolve, but it'd really just be adding to our run time w/o benefit.

Reply:
Yeah, I get that; I'm saying that I'd like to see that explanation put in a comment in this file next to this configuration setting, so that it's clear to any future developer why we have these values and not other values in the setting.